README.md

```diff
@@ -61,6 +61,41 @@ optimizt --webp path/to/directory
 optimizt `find . -type f -name '*.jpg'`
 ```
 
+## About “lossy” and “lossless”
+
+### JPEG
+
+In lossy mode [jpegoptim](https://github.com/tjko/jpegoptim) is used with flags: `--strip-all`,
+`--all-progressive`, `--max=80`.
+
+Lossless mode uses a [Guetzli](https://github.com/google/guetzli) encoder with `--quality 90` flag.
+
+Guetzli is aims for excellent compression density at high visual quality.
+
+If you re-optimize the same file in lossless mode, the file size may decrease, but the visual quality will also degrade.
+
+### PNG
+
+For both modes, [pngquant](https://github.com/kornelski/pngquant) (lossy compressor) is used with flags:
+`--speed 1`, `--strip`.
+
+### GIF
+
+For both modes, [gifsicle](https://github.com/kohler/gifsicle) utility is used.
```

Comment:
I think that we don't need comma in this sentence. In the last sentence of this section neither.